web/web.go

```diff
@@ -224,9 +225,11 @@ func (h *Handler) Quit() <-chan struct{} {
 	return h.quitCh
 }
 
-// Reload returns the receive-only channel that signals configuration reload requests.
-func (h *Handler) Reload() <-chan struct{} {
-	return h.reloadCh
+// Reload returns two cahnnels:
+//   - receive-only channel that signals configuration reload requests.
+//   - send-only channel with errors happened during configuration reload.
```

Comment:
What if I'll create `Reload` function inside `cmd/main.go` which will return error explicitly and will encapsulate all the logic inside it and just call it from the handler?